db/schema.rb

```diff
@@ -24,14 +24,6 @@
     t.string "image"
   end
 
-  create_table "salads", force: :cascade do |t|
-    t.string "name"
-    t.integer "price"
-    t.string "image"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
```

Comment:
You never have to edit the schema directly. Just run the migrations. Not a blocker though.